src/EnergyPlus/DXCoils.cc

```diff
@@ -614,8 +614,8 @@ namespace EnergyPlus::DXCoils {
 
                     // Determine combined performance
                     state.dataDXCoils->DXCoil(DXCoilNum).OutletAirEnthalpy =
-                        (1.0 - S12RuntimeFraction) * S1OutletAirEnthalpy + S12RuntimeFraction * S12OutletAirEnthalpy;
-                    state.dataDXCoils->DXCoil(DXCoilNum).OutletAirHumRat = (1.0 - S12RuntimeFraction) * S1OutletAirHumRat + S12RuntimeFraction * S12OutletAirHumRat;
+                        (1.0 - S2PLR) * S1OutletAirEnthalpy + S2PLR * S12OutletAirEnthalpy;
+                    state.dataDXCoils->DXCoil(DXCoilNum).OutletAirHumRat = (1.0 - S2PLR) * S1OutletAirHumRat + S2PLR * S12OutletAirHumRat;
```

Comment:
Use `PLR`, not `RuntimeFraction` to calculate leaving conditions for 2-stage operation (Coil:Cooling:DX:TwoStageWithHumidityControlMode). `RuntimeFraction` is appropriate for power consumption.

Reply:
Agreed.